topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md

```diff
@@ -104,9 +104,9 @@ Let's start with a fresh history.
 > 1. Download the list of peak regions (the file [`GSE37268_mof3.out.hpeak.txt.gz`](https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE37268&format=file&file=GSE37268%5Fmof3%2Eout%2Ehpeak%2Etxt%2Egz)) from [GEO](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37268) to your computer
 > 2. Click on the upload button in the upper left ot the interface
 >
->    ![Upload icon](../../../galaxy-interface/images/upload_button.png)
+>![Upload Data Icon](../../images/Upload_Data.PNG)
```

Comment:
1. Here you've uploaded a new image, so now we have two images in the GTN: the old and the new. The old is still possibly used in other tutorials, and your new one is only used here, which is less than optimal. Could you please overwrite the old image, with your new image file? (I.e. move `Upload_Data.PNG` to overwrite/replace `galaxy-interface/images/upload_button.png`)
2. Please re-add the space at the start, it's important for the numbering. By having this all the way to the left, markdown interprets the 3 below as starting a new list and it will be numbered 1 instead of 3 as expected.